### DISCUSSION TITLE
zh/guide/drivers/mega

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#208] [Announcements]
# zh/guide/drivers/mega

MEGA官网：https://mega.nz/login 邮箱 MEGA登录账号 密码 登录密码 Tips 默认本地代理，需要proxy 默认使用的下载方式

https://alist.nn.ci/zh/guide/drivers/mega.html

**Comment by bsgn2023** [comment]
有VPN，代理地址应该怎么填

**Comment by zhlx2835** [comment]
连不上，请求超时

**Comment by KomiyaTaichi** [comment]
始終無法添加

**Reply by KomiyaTaichi** [reply]
顯示超時

**Comment by phytomorphogenesis** [comment]
共享出去的文件夹就不加载了怎么解决鸭

**Comment by thetesttoy** [comment]
set http_proxy=http://127.0.0.1:10809
set https_proxy=http://127.0.0.1:10809
set no_proxy=localhost,127.0.0.1
alist server 显示超时的可以试试上面命令，顺便推一下本人网站loliacg.top欢迎来玩
